impress-org/givewp
# Refactor: move legacy donation forms into v2 domain

## Description



This moves all references to the `Give\DonationForms` domain into `Give\DonationForms\V2` for the purpose of making room for v3 forms that will live in the root of that domain.

## Affects



Legacy donation forms & list tables.

## Visuals



N/A

## Testing Instructions



Just make sure legacy forms are working as expected and no add-ons are using those old domains

## Pre-review Checklist
